expression-language/4.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta Expression Language 4.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Expression Language 4.0"
+date: 2020-07-16
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Expression Language defines an expression language for Java applications.
 
 * [Jakarta Expression Language 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/4.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0-RC2.pdf) (PDF)
-* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0-RC2.html) (HTML)
+* [Jakarta Expression Language 4.0 Specification Document](./jakarta-expression-language-spec-4.0.pdf) (PDF)
+* [Jakarta Expression Language 4.0 Specification Document](./jakarta-expression-language-spec-4.0.html) (HTML)
 * [Jakarta Expression Language 4.0 Javadoc](./apidocs)
-* [Jakarta Expression Language 4.0 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/master/nightly/expression-language-tck-4.0.0.zip)([sig](),[sha](),[pub]())
+* [Jakarta Expression Language 4.0 TCK](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.0.zip)([sig](),[sha](),[pub]())
```

Comment:
The [sig], [sha], and [pub] urls need to be filled in.  Ensure these are corrected before merging, but I wouldn't hold up the vote for these corrections.

Reply:
I've made the change myself, @kwsutter does that look better?